test/examples/main_test.go

```diff
@@ -0,0 +1,11 @@
+package examples_test
```

Comment:
do not use underscores: https://golang.org/doc/effective_go.html#package-names

Reply:
Question: can't `golangci-lint` or another linter catch those for us?